Shared/UI/Transitions/Transition Types/TransitionRouter+Blur.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  TransitionRouter+CrossDissolve.swift
+//  Jibber
+//
+//  Created by Martin Young on 4/25/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import UIKit
+
+extension TransitionRouter {
+
+    /// For presentation, blurs the fromVC then fades in the toVC.
+    /// For dismiss, fades out fromVC and unblurs the toVC.
+    func blur(transitionContext: UIViewControllerContextTransitioning) {
```

Comment:
I made a new transition function to blur the background and fade in the presented vc.